src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,16 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.1.0-preview</Version>
+		<Version>19.2.0-preview</Version>
```

Comment:
@number213 @MikhailTryakhov what are your plans regarding releasing a stable version

Reply:
@dsgouda yes.
Can we also start deployment process to other SDKs as well to be synced

Reply:
Not sure what that means, you will have to talk to individual SDK teams regarding this